### DISCUSSION TITLE
How can I use influxQL for influxdb Scalers

### DISCUSSION THREAD

**Discussion by essong** [kedacore/keda#3651] [Q&A / Need Help]
Hello, I'm running the latest version of keda
But, Because of influxdb spec. I use influxQL in InfluxDbScalers.
How can I use influxQL for influxdbScaler. 
And What is the minimum version of influxdb supported by KEDA?

**Comment by JorTurFer** [comment]
Hello, 
I think there isn't any minimum version, we use the influx SDK to make the queries. I can't see anything related
